_posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Lesson Infrastructure Subcommittee 2018 February meeting"
+date: 2018-02-28
+time: "10:00:00"
+category: ["Community", "Lesson Infrastructure Subcommittee"]
+---
+
+On 14 February 2018 at 18:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2018 February meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+Software Carpentry and Data Carpentry had merge.
+Their [Executive Council was elected](https://software-carpentry.org/blog/2017/12/executive-council-2018.html),
+they have a [new fiscal sponsor](https://software-carpentry.org/blog/2018/01/fiscal-sponsor-transition.html)
+and
+a [new logo](https://software-carpentry.org/blog/2018/02/our-new-logo.html).
+Also,
+they have [hired Dr. François Michonneau](http://www.datacarpentry.org/blog/curriculum-dev-scaling/) to lead curriculum development efforts.
+
+This all are very exciting news. And we even didn't finished February.
+
+## Windows Installer
+
+We will proceed to adopt Git for Windows v2.15.1,
+which includes nano,
+and drop our Windows installer.
+This should simplify our installation instructions
+and
+provide a better experience for our learners during the workshop.
+All the details of this changes are covered in [this document](https://github.com/carpentries/lesson-infrastructure/blob/master/liep/001-git-for-windows-nano.md)
+that summarise many pull requests and issues over the last 6 months.
+
+## Workshop Installation Instructions
+
+In January,
+a [blog post was published](https://software-carpentry.org/blog/2018/01/proposal-workshop-template.html)
+with a proposal to enhancement the process to customize the installation instructions at the workshop page.
```

Comment:
to enhance the process of customizing the installation instructions on the workshop page.